### ISSUE TITLE
The internal dashboard should also specify monospace font

### ISSUE THREAD

**Issue by msoxzw** [uBlockOrigin/uBlock-issues#2457]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

The dashboard has not specified which monospace font to use, so the actual font is determined by language, browser, and OS, which is particularly nasty for CJK language.

### A specific URL where the issue occurs.

Internal URL

### Steps to Reproduce

Open "My rules" pane on the dashboard:
https://github.com/gorhill/uBlock/wiki/Dashboard:-My-rules

### Expected behavior

The monospace font is expected to be consistent regardless of language, browser, and OS.

### Actual behavior

The monospace font varies depending on language, browser, and OS.

### uBO version

1.46.0

### Browser name and version

Firefox 108

### Operating System and version

Windows 10 22H2

**Comment by gothic-bum** [comment]
https://github.com/gorhill/uBlock/wiki/Advanced-settings#uistyles

**Comment by gwarser** [comment]
How bad it looks? Screenshot?

Which font should be best to use?

**Comment by krystian3w** [comment]
IMO follow user config in browser/OS is not bad.

**Comment by wallrik** [comment]
The [default monospace font](https://user-images.githubusercontent.com/10890923/212566634-8218dde3-f711-4143-9206-33a54a565479.png) on Windows seem to be `Consolas`. But looking at something like Japanese, or Simplified Chinese, there is indeed a different default font. But I would think that's a good thing, no?

Anyway, the style GitHub uses currently looks like this:
```css
tt, code, samp {
 font-family:ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace;
 font-size:12px
}
```

**Comment by krystian3w** [comment]
Looks as disabled ClearType in Windows 10?

**Comment by gorhill** [comment]
The font family for the CodeMirror widget is set to `monospace`. This means is up to the browser to pick the monospace font as configured by the user. I am not understanding why it's a uBO issue. Using `monospace` means uBO does not have to guess whichever specific (and potentially proprietary) font family for `monospace` is best on any OS.

**Comment by gwarser** [comment]
What font do you have set here: 
![image](https://user-images.githubusercontent.com/886325/216108679-10c30610-a74d-421b-aa75-8fee794270fa.png)

**Comment by gorhill** [comment]
Right, on my side I had `Default (DejaVu Sans Mono)` and I changed it to `Noto Sans Mono` because I preferred this. So this should really be left to user settings from browser/OS.

**Comment by Thorin-Oakenpants** [comment]
This was actually an issue in Tor Browser, where between ESR releases, the mac monospace changed from whatever to Menlo, and Menlo was blocked (TB uses whitelisting on macs). You can get the same effect when the default font is not present on the system or not available for whatever reason (users might delete the font, or it's corrupted, or set a different font which is subsequently removed). And thus monospace wasn't even being used.

long story short - specify font fallback and save the footgun: I suggest
- mac:  `monospace, Menlo, Courier, "Courier New", Monaco`
- windows: `monospace, Consolas, Courier, "Courier New", "Lucida Console"`
- linux: no idea, have fun

This still leaves the app langauge setting to decide + use the default monospace

**Comment by krystian3w** [comment]
Are you sure this order is good in CSS?

I mean did you do a font extermination test on both systems and the browser let you jump to the next one instead of reaching for e.g. sans-serif like Times New Roman (due didn't detect 4-5 as monospace better than random).

**Comment by msoxzw** [comment]
This issue might be regarded to be resolved if the author @gorhill opines that monospace fonts should be configured by use via browser setting. Additionally, I am reluctant to modify global fonts preferences due to font fingerprinting.
